handlers/bulk_refunds.go

```diff
@@ -95,6 +95,29 @@ func HandleGovPayBulkRefund(w http.ResponseWriter, req *http.Request) {
 	w.WriteHeader(http.StatusCreated)
 }
 
+// HandleProcessPendingRefunds retrieves a list of payments in the DB with
+// a refund-pending status and fires a request to the payment provider to
+// issue a refund
+func HandleProcessPendingRefunds(w http.ResponseWriter, req *http.Request) {
```

Comment:
should there be a TODO comment in this func to denote where the request to the payment provider will occur?

Reply:
Resolving as the next story is addressing this